packages/core/types/index.js

```diff
@@ -645,6 +645,28 @@ export type ASTGenerator = {|
 
 export type BundleBehavior = 'inline' | 'isolated';
 
+export type ParcelTransformOptions = {|
+  filePath: FilePath,
+  code?: string,
+  env?: EnvironmentOptions,
+  pipeline?: ?string,
```

Comment:
Not sure if pipeline makes sense as a public option? Shouldn't this be inferred from the `.parcelrc`?

Reply:
I am happy to remove this `pipline` property.

Reply:
removed.